tests/lib/openstack_api/test_openstack_service.py

```diff
@@ -0,0 +1,106 @@
+from unittest.mock import MagicMock, NonCallableMagicMock
+from openstack_api.openstack_service import disable_service, enable_service
+
+
+def test_disable_service_function_disables_service_when_enabled():
```

Comment:
I would keep your fuction names short and leave the description to be in the doc string
```suggestion
def test_disable_service():
```

Reply:
Another conversation I had about unit tests with Anish where the reason we put a long description in the function title, is to easier see locate what function in an error log.

In this case it might be difficult to differentiate the two function names as one tests when the service is already enabled and the other tests when it's not.

Let me know your thoughts.

Reply:
If it makes working on code easier for you and others than I would say keep it. No harm either ways